graphql/availability.graphql

```diff
@@ -0,0 +1,38 @@
+extend type Query {
+    availabilities: [Availability!]! @paginate
+    availabilityByDate(from: Date! @where(clause: "whereDate")): [Availability!]! @all
+    availabilityBetweenDates(from: DateRange @whereBetweenDate): [Availability!]! @all
```

Comment:
the discussion above has also made me think that these aren't really usefull? It would be better to have options to show, say, availability on a date for a specific user

Reply:
I've removed them to get the story added, we can think about adding it at a later date when it's needed.